Docs/sdk/assemblies-for-multiple-vs-versions.md

```diff
@@ -0,0 +1,31 @@
+## Shipping Windows Forms design-time assemblies for multiple Visual Studio versions in a single NuGet package
+
+You may find that you need to ship more than one version of your design-time assemblies, each targeting a different version of Visual Studio.  For example, you might have basic design-time support for your controls starting in VS 17.6, but due to the introduction of some new designer extensibility APIs in 17.7 etc. you might be able to provide a better user experience in 17.8 and later VS versions. In that case you would want to ship two sets of design-time assemblies, one for Visual Studio version 17.6 and the other for 17.8 in a single NuGet package.  
+
+This feature to support design-time assemblies targeting multiple Visual Studio versions in a single NuGet package is supported from 17.0 onwards.
+
+#### NuGet package structure supporting design-time assemblies targeting multiple Visual Studio versions:
+
+![NuGet package structure supporting design-time assemblies targeting multiple Visual Studio versions](/docs/sdk/images/multi-vs-versions-nuget-structure.png)
```

Comment:
Paths to links need to be case sensitive. If possible, its better if we change directory names to lowercase.

Reply:
They only need to be case sensitive for non-Windows platforms.

Reply:
Were you able to open the image form link here? on my machine (Windows) forcing me the URL being case sensitive. So `docs `in URL needed to be `Docs`. If we are not changing folder names in this PR, we should make sure URLs case sensitive.

Reply:
Everything is previewing just fine on my machine with the PR as is. The folders I did not change are the ones in all of Klaus's solutions - that would be a pain. But docs\sdk is all lower case.

Reply:
> Were you able to open the image form link here? on my machine (Windows) forcing me the URL being case sensitive. So `docs `in URL needed to be `Docs`. If we are not changing folder names in this PR, we should make sure URLs case sensitive.


> Everything is previewing just fine on my machine with the PR as is. The folders I did not change are the ones in all of Klaus's solutions - that would be a pain. But docs\sdk is all lower case.

You can preview the docs here: https://github.com/microsoft/winforms-designer-extensibility/blob/McGawNugetDocs/Docs/sdk/assemblies-for-multiple-vs-versions.md

The image isn't showing up.

In general, I suggest keeping all folders and filenames in the lower case.

Reply:
That was my intent, I thought I had changed docs to lower case - maybe I didn't push that commit yet, because you're right, I don't see it either

Reply:
OK, this is working now.